content/cookbook/wallets/create-keypair.md

```diff
@@ -11,7 +11,7 @@ are [connecting to a wallet](/content/cookbook/wallets/connect-wallet-react),
 you do not need to worry about the keypair. Otherwise a keypair must be
 generated for signing transactions.
 
-```javascript
+```javascript file=/code/content/cookbook/wallets/create-keypair.ts#L1-L3
```

Comment:
Can you double check the content displays correctly on solana-com. It looks like if you don't include a filename for a snippet, the provided file path gets used.

![CleanShot 2024-10-25 at 10 28 20@2x](https://github.com/user-attachments/assets/3a1bf538-d317-4876-8f71-df9b3d4defd1)


Also, why is there a file path, but the code block still contains the snippet?

````
```javascript file=/code/content/cookbook/wallets/create-keypair.ts#L1-L3
import { Keypair } from "@solana/web3.js";

const keypair = Keypair.generate();
```
````

Reply:
Code block has content for crowdin support i guess.

Reply:
This should be a PR on [solana-com](https://github.com/solana-foundation/solana-com)?